app/assets/javascripts/components/nav_signals/active_signals.js.jsx

```diff
@@ -0,0 +1,37 @@
+var ActiveSignalPanel = React.createClass({
+  render: function() {
+    return (
+      <div>
+        {this.renderPanel()}
+      </div>
+    );
+  },
+
+  renderPanel: function() {
+    signals = [];
+    this.props.signals.forEach(function(s, idx) {
+      signals.push(
+        <div className={'panel signal-panel '+ s.signal_type}>
+          <SignalIcon path={'icons/' + s.signal_type} />
+          <div className='panel-header'>
+            {'# ' + s.name}
+          </div>
+          <div className='panel-body'>
+            Send your users a special offer every time they send a custom hashtag
+          </div>
+          <div className='panel-status'>
+            <div className='signal-status'>
+              <div className='circle active'>
+                <span className='status'>ACTIVE</span>
```

Comment:
We might need to start thinking about building out API endpoints for these components to hit. We're really living in a client-side world here, so it would make sense if we build out these objects using JSON.